### DISCUSSION TITLE
SMB连接，failed get objs: failed to list objs: open *: file does not exist

### DISCUSSION THREAD

**Discussion by jesongit** [AlistGo/alist#2979] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/29886847/211351113-10bbb6b6-ceb0-442d-b5ef-9a29602fbb78.png)
情况如上图所示
在TG群询问，3.7和3.8的SMB挂载有有问题，应该会下个版本修复，给遇到问题的人说明一下

**Comment by jesongit** [comment, marked as answer]
另外指路一个[SMB配置图解](https://github.com/alist-org/alist/discussions/2215#discussioncomment-4104018)，文档没有相关说明，感谢这位老哥

**Reply by oldking139** [reply]
试着将 _根文件夹路径_ 设定为:```/```

我也是参照这个哥们的截图，不过没管用。改成```/```就好了。

**Reply by shanhai2333** [reply]
> 试着将 _根文件夹路径_ 设定为:`/`
> 
> 我也是参照这个哥们的截图，不过没管用。改成`/`就好了。

牛逼，可以了

**Comment by shanhai2333** [comment]
这样哦，我还以为怎么了
